.devcontainer/base-dev/base-dev.Dockerfile

```diff
@@ -1,4 +1,177 @@
-FROM ghcr.io/ubcsailbot/sailbot_workspace/pre-base:ros_humble-ompl_4c86b2f-mongo_367-v2 as base
```

Comment:
I see that you combined pre-base and base-dev images. So in the future, when members add a dependency to the base-dev image, the workflow to build the image will install ompl and ros each time right?

Reply:
Technically yes, but in reality docker caches each command in the dockerfile (creates a layer for each command). So ideally Docker will start executing commands at the first line that you change.